parking-spot-submodule/src/main/java/ua/lviv/iot/application/proto/converter/ParkingSpotConverter.kt

```diff
@@ -1,9 +1,9 @@
-package ua.lviv.iot.parkingServer.converter
+package ua.lviv.iot.application.proto.converter
```

Comment:
converter knows about proto and proto - is infra component, so converter should not be in application package

Reply:
✅